### ISSUE TITLE
translation string missing for entry tabpage "general"

### ISSUE THREAD

**Issue by the-dino-girl** [JabRef/jabref#10325]
Hi

Sorry to bring this up again but could you check if the "general" string can't be translated? I think it can because it's not a field and the other tabpages are translated as well. I can't find any translation string for it (as you can see it's still English)

https://user-images.githubusercontent.com/143431619/265235177-1b49a663-6865-4bd6-9440-52ed01d8404a.png

**Comment by Siedlerchr** [comment]
The general comes from here: You can rename the tab (requires a restart of JabRef) 
<img width="897" alt="grafik" src="https://github.com/JabRef/jabref/assets/320228/928a2c8b-2e75-465d-9bb4-c1db22b48288">

e.g. renamed to test:
<img width="876" alt="grafik" src="https://github.com/JabRef/jabref/assets/320228/374cd90b-1176-464c-a0b6-1d38c49c39ac">

**Comment by the-dino-girl** [comment]
> No problems, JabRef has so many features, there are corners in the program I only barely touched and in the code as wel^^.

If I don't export the settings to XML file, they are gone when I update jabref?

**Comment by Siedlerchr** [comment]
No, settings are kept intact and are migrated to the new version when old ones change

**Comment by the-dino-girl** [comment]
So the XML file is to transfer settings between devices?
